src/ios/AssetBundleManager.swift

```diff
@@ -220,7 +220,15 @@ final class AssetBundleManager: AssetBundleDownloaderDelegate {
       // If we find a cached asset, we make a hard link to it
       if let cachedAsset = cachedAssetForAsset(asset) {
         do {
-          try fileManager.linkItem(at: cachedAsset.fileURL as URL, to: asset.fileURL as URL)
+          // Workaround for https://github.com/meteor/cordova-plugin-meteor-webapp/issues/56
+          //try fileManager.linkItem(at: cachedAsset.fileURL as URL, to: asset.fileURL as URL)
+          let linkExists = (try? asset.fileURL.checkResourceIsReachable()) ?? false
+          if !linkExists {
+            try fileManager.linkItem(at: cachedAsset.fileURL as URL, to: asset.fileURL as URL)
```

Comment:
Likewise, I think we could try the `fileManager.linkItem` operation, and if it fails just remove the existing link at `asset.fileURL` and try again.